packages/media/src/generateVideoStrip.js

```diff
@@ -74,8 +74,8 @@ async function generateVideoStrip(element, resource, stripWidth, stripHeight) {
   // Calculate the offset between each frame to be grabbed
   const frameDistance = length / (frameCount - 1);
 
-  // Preload video and adjust size
-  const video = await preloadVideo(src);
+  // Preload video and adjust size. Skipping the first ms to prevent blank frames in chrome.
+  const video = await preloadVideo(`${src}#t=0.000001`);
```

Comment:
Any advantage of this or general difference vs. the following?

```js
const video = await preloadVideo(src);
await seekVideo(video, 0.000001);
```

Reply:
None that I can see. This looks much cleaner and readable. Will update that part